### DISCUSSION TITLE
Resolve Exports currently defaults to False

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#637] [Problem]
Right now, `idom.web.module_from_template`'s `resolve_exports` currently defaults to `False` when not in `IDOM_DEBUG_MODE`, which can lead to an awkward developer experience. We shouldn't have to rely on `IDOM_DEBUG_MODE` to let the user know of something that is confirmable to be a typo.

The issue is predominantly that, when in a barebones configuration, the user has no Python logs telling whether something they've has failed to export. Even when attempting to render a failed export, there are no logs/console output given that something has went wrong. We definitely should warn when a module confirmably does not exist within a properly fetched esm.sh package

We should pick one of these suggestions to remediate this
1) Remove the `resolve_exports` option. Always resolve exports and log an error upon failed export rather than raising an exception.
2) Always default `resolve_exports` to `True`
3) Provide an error log upon attempting to render an export that has failed to resolve

Opening this as a discussion first so we can talk about whether this should be addressed.

**Comment by rmorshea** [comment]
For option 3, there should be a client-side log, but that's easy for people to miss. We have an issue that captures this though: https://github.com/idom-team/idom/issues/346

**Reply by Archmonger** [reply]
Yeah it is easy to miss. Most Python devs are solely looking at the Python terminal.

Personally, option 1 is my preferred choice.

**Reply by Archmonger** [reply]
So just want to check in - Does this discussion belong in an issue ticket?

This is QOL, so would be something to look at in V2
